.github/config.yml

```diff
@@ -0,0 +1,27 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
```

Comment:
Remove this config file as it is already added by someone in a pull request then it can create conflicts.

Reply:
@kritikaparmar-programmer thats just  a comment

Reply:
I have updated it anyways ☺